pdns/recursor_cache.cc

```diff
@@ -19,23 +19,22 @@ MemRecursorCache::MemRecursorCache(size_t mapsCount) : d_maps(mapsCount)
 {
 }
 
-size_t MemRecursorCache::size()
+size_t MemRecursorCache::size() const
```

Comment:
There is trade-off to be considered, since having a single atomic counter would increase the number of times that a cache line has to be cleared. Unless I'm mistaken we don't call `size()` too often, so perhaps the current behaviour is better?